src/style.css

```
@@ -1,5 +1,8 @@
 @import 'tailwindcss';
 
+/* dark: variant를 미디어쿼리 대신 .dark 클래스 기반으로 설정 */
+@variant dark (&:where(.dark, .dark *));
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)